applications/typechain-polkadot-follow-up.md

```diff
@@ -157,7 +157,7 @@ We have decided to describe a full roadmap of a Typechain here, with estimates.
 | ------ | ----------- | ------------- |
 | 0a     | License  | MIT |
 | 0b     | Documentation | We will provide TypeScript code examples of this package in use, inline documentation, JSDoc, and the description of its features. |
-| 1 | TS Types | We will research & match types from ABI to TypeScript, compatible with polkadot{.js} v8 library. Separately, for methods' arguments and return values. Files with types definition will be generated. | 
+| 1 | TS Types | We will research & match types from ABI to TypeScript, compatible with polkadot\{.js} v8 library. Separately, for methods' arguments and return values. Files with types definition will be generated. | 
```

Comment:
No need to escape `}`?

Reply:
looks the same and works 🤷 . So guess it's not necessary.